features/search/default-queries-uitid.feature

```diff
@@ -0,0 +1,66 @@
+@sapi3
+Feature: Test the Search API v3 default queries from UiTID
+
+  Background:
+    Given I am using the UDB3 base URL
+    And I am using an UiTID v1 API key of consumer "uitdatabank"
+    And I am authorized as JWT provider v1 user "centraal_beheerder"
+    And I send and accept "application/json"
+
+  Scenario: Search for a place blocked by the default query
+    Given I create a place from "places/hemmekes.json" and save the "id" as "placeId"
+    And I am using the Search API v3 base URL
+    And I am not authorized
+    And I am using an UiTID v1 API key of consumer "sapi3KeyWithUitIdFilterForDiest"
```

Comment:
The default query that gets applied here is a bit hidden, maybe we need to put it in a comment somewhere? Now it's a bit hard to figure out why the place gets filtered out

Reply:
I have put it in this file, alternative might be to put it in comments in `config.features.php`